app/code/core/Mage/Adminhtml/Block/Sales/Order/Create/Form/Address.php

```diff
@@ -37,12 +37,22 @@ class Mage_Adminhtml_Block_Sales_Order_Create_Form_Address extends Mage_Adminhtm
 
     /**
      * Return Customer Address Collection as array
+     * Ignore addresses when country is not authorized
```

Comment:
This is a better comment

**Ignore addresses when the country is not allowed**